src/components/PDFView/index.js

```diff
@@ -52,6 +53,14 @@ class PDFView extends Component {
 
         const workerBlob = new Blob([pdfWorkerSource], {type: 'text/javascript'});
         pdfjs.GlobalWorkerOptions.workerSrc = URL.createObjectURL(workerBlob);
+        this.setMaxCanvaSizeInState = this.setMaxCanvaSizeInState.bind(this);
+        this.setMaxCanvaHeightInState = this.setMaxCanvaHeightInState.bind(this);
+        canvasSize.maxArea({
+            onSuccess: this.setMaxCanvaSizeInState 
+         });
+         canvasSize.maxHeight({
+            onSuccess: this.setMaxCanvaHeightInState 
```

Comment:
```suggestion
            onSuccess: (height) => this.state.maxCanvasHeight =  height;
```

Do we need to store it in state? Can we just do this.maxCanvasHeight?